### DISCUSSION TITLE
Bad server/bad connection alerts

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#906] [Q&A] [answered]
Hi Dsys!

When user try to submit a form by hitting the submit button (There is no error on the form), then there's a bad server/bad connection error that prevents them from continue, what kind of alerts do we show here?

Should we be using the alert banner or using a whole page to show this error (something like "502 gateway error" on the web page)? User can only refresh the page to solve this issue. Would like to hear some guidance.

Thanks,

Charlene

**Comment by corinne-nagel** [comment]
Hi @charleneyjh1 I'll take this to the team and get back to you with some ideas. Currently our time to respond is a few days. Thanks!

**Comment by corinne-nagel** [comment, marked as answer]
Hi @charleneyjh1 I think this article will give you some great guidance on what alert method to choose: https://www.nngroup.com/articles/indicators-validations-notifications/ I'd also suggest you test whatever method you decide to go with.

In the meantime, could you please provide a couple mocks of what you were thinking? That will help us give you better direction, thanks!

**Reply by charleneyjh1** [reply]
Thanks Corinne! We've decided to use Alerts for that.
